README.md

```diff
@@ -3,6 +3,7 @@
 [![Build Status](https://travis-ci.org/peridot-php/peridot.png)](https://travis-ci.org/peridot-php/peridot) [![HHVM Status](http://hhvm.h4cc.de/badge/peridot-php/peridot.svg)](http://hhvm.h4cc.de/package/peridot-php/peridot)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/peridot-php/peridot/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/peridot-php/peridot/?branch=master)
 [![Packagist](https://img.shields.io/packagist/dt/peridot-php/peridot.svg?style=flat)](https://packagist.org/packages/peridot-php/peridot)
+[![codecov](https://codecov.io/gh/peridot-php/peridot/branch/master/graph/badge.svg)](https://codecov.io/gh/peridot-php/peridot)
```

Comment:
Couple of suggestions:

- We can make all of these badges look consistent by using [shields.io badges](http://shields.io/). They support just about everything, and are pretty quick to respond to PRs if you need something not already on there. Take a look at how I do [Phony's badges](https://github.com/eloquent/phony/blob/master/README.md).
- The "HHVM Status" badge, both the one you have here, and the one on shields.io, show an incorrect status if you use a newer Travis config for HHVM versions like we've added here. I raised the issue with the creator here: h4cc/hhvm-progress#61, but there's been no fix yet. I made my own "fake" badge using shields.io, until the problem gets addressed. You'll also see that in Phony's README.